localtypings/pxtcustomeditor.d.ts

```diff
@@ -0,0 +1,186 @@
+declare namespace pxt.extension {
+    interface Message {
+        id?: string;
+        type: "request" | "response" | "event";
```

Comment:
How does this relate to https://github.com/Microsoft/pxt/pull/2704/commits/1fca35f12c6038ed353712b603b8d97d4e1860cf ?